### PR TITLE
added name to team members

### PR DESCRIPTION
**Pull request by TaylorDJones11**

### CODE REVIEW

**Review by annonymousauthority — DISMISSED** [review]
LGTM

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

**Review by annonymousauthority — APPROVED** [review]
Nice! LG

File: wireframes/Add Community Page.png
Comment (msarit) [inline wireframes/Add Community Page.png:1]: Let's have these wireframe images merged as part of a separate pull request @TaylorDJones11 .
Take them out of this current PR.

These images are 🙌🏾 🙌🏾 🙌🏾

**Review by msarit — CHANGES_REQUESTED** [review]

**Comment by msarit** [comment]
@TaylorDJones11 Fix the conflicts Taylor then I will merge 🙏🏾

**Comment by TaylorDJones11** [comment]
I made the changes to repo

**Review by msarit — APPROVED** [review]

**Pull request merged**